lib/eclipse/EclipseState/Runspec.cpp

```diff
@@ -52,12 +54,13 @@ std::ostream& operator<<( std::ostream& stream, const Phase& p ) {
 
 using un = std::underlying_type< Phase >::type;
 
-Phases::Phases( bool oil, bool gas, bool wat, bool sol, bool pol ) noexcept :
+Phases::Phases( bool oil, bool gas, bool wat, bool sol, bool pol, bool energy ) noexcept :
     bits( (oil ? (1 << static_cast< un >( Phase::OIL ) )     : 0) |
           (gas ? (1 << static_cast< un >( Phase::GAS ) )     : 0) |
           (wat ? (1 << static_cast< un >( Phase::WATER ) )   : 0) |
           (sol ? (1 << static_cast< un >( Phase::SOLVENT ) ) : 0) |
-          (pol ? (1 << static_cast< un >( Phase::POLYMER ) ) : 0) )
+          (pol ? (1 << static_cast< un >( Phase::POLYMER ) ) : 0) |
```

Comment:
That we treat energy as a phase must be the extended phase concept :-)

Reply:
the same applies to polymer and solvent, though. see the commit comment for this:

https://github.com/OPM/opm-parser/pull/1158/commits/1703ad5cb00875314b11da381d103a897d229620

Reply:
I agree with the commit comment, that ConservedQuantities is more precise. Not sure that it is practical/wanted/needed to switch though. I'm sure that would lead to different annoyances!